<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TQConnectWidget</class>
 <widget class="QWidget" name="TQConnectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="4" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Пользователь БД</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Пароль БД</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Сервер</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Источник</string>
          </property>
          <property name="buddy">
           <cstring>dbmsEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Проект</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Вид</string>
          </property>
          <property name="buddy">
           <cstring>dbClassComboBox</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QLineEdit" name="projectEdit"/>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QComboBox" name="dbClassComboBox"/>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="sqlUserEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QToolButton" name="btnProjects">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="sqlPassEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLineEdit" name="serverEdit"/>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QCheckBox" name="trustedUserBox">
          <property name="text">
           <string>Авторизация Windows</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="btnDBMS">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="dbmsEdit"/>
        </item>
        <item row="9" column="1" colspan="3">
         <widget class="QCheckBox" name="autoLoginCheck">
          <property name="text">
           <string>Соединяться автоматически при входе</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="passwordEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="userEdit"/>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="btnOpen">
          <property name="text">
           <string>Открыть</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>19</number>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Добавить...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnDel">
       <property name="text">
        <string>Удалить...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbClassComboBox</tabstop>
  <tabstop>dbmsEdit</tabstop>
  <tabstop>btnDBMS</tabstop>
  <tabstop>serverEdit</tabstop>
  <tabstop>trustedUserBox</tabstop>
  <tabstop>sqlUserEdit</tabstop>
  <tabstop>sqlPassEdit</tabstop>
  <tabstop>projectEdit</tabstop>
  <tabstop>btnProjects</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>autoLoginCheck</tabstop>
  <tabstop>btnOpen</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDel</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
